go.mod

```
@@ -0,0 +1,12 @@
+module pas
```

```suggestion
module github.com/putdotio/pas
```

Public repo oldugu icin modul adini `github.com/putdotio/pas` yapalim. Boylece import'lari da degistirmemize gerek olmaz.